src/discordTools/discordEmbeds.js

```diff
@@ -719,18 +753,20 @@ module.exports = {
             fields: [
                 { name: playersFieldName, value: `\`${rustplus.getCommandPop(true)}\``, inline: true },
                 { name: timeFieldName, value: `\`${time[0]}\``, inline: true },
-                { name: wipeFieldName, value: `\`${rustplus.getCommandWipe(true)}\``, inline: true }],
+                time[1] ? { name: timeLeftTitle, value: `\`${time[1]}\``, inline: true } : emptyFieldValue,
+            ],
             timestamp: true
         });
 
         if (time[1] !== null) {
```

Comment:
Why is this dependent on time?

Reply:
I switched field `Time till` with `Wipe` field so wipe info is on second row and `Players`, `Time` and `Time till` are on first row. So this row you pointed is field `Time till` (if you compare actual bot with images i add to pull request you will see). 

If you want it in different order i can fix it however you like.

Reply:
What I dont understand is why you write out wipe stuff only if time[1] is present...

This part:

`        if (time[1] !== null) {
            embed.addFields(
                { name: wipeFieldName, value: `\`${rustplus.getCommandWipe(true)}\``, inline: true },
                firstWipe,
                secondWipe,
                );
        }
        else {
            embed.addFields(emptyFieldValue);
        }`